src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -169,3 +178,63 @@ def train_once(self, itr, paths):
 
         """
         raise NotImplementedError
+
+    def _obtain_evaluation_samples(self, batch_size=1000):
```

Comment:
the evaluation trajectories must be whole (have `done=True`) or reach `max_path_length`, so I don't think `batch_size` is a useful parameter here. Plus, you use `batch_size` == `max_path_length`, which I think is confusing.

I think the signature of this function should be `_obtain_evaluation_samples(self, num_trajs=100)`. It collects `num_trajs` evaluation trajectories which are either terminated or length `self.max_path_length`. I think `num_trajs = 100` is a sane default. 10 trajectories is kind of low. If 100 is too slow, I recommend going with 30.